.github/workflows/release.yaml

```diff
@@ -37,43 +34,45 @@ jobs:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
 
   publishdocker:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Why shift to `ubuntu-24.04` for some and `ubuntu-latest` for others? Should probably keep it consistent.

Note that on fabric core we shifted everything to `ubuntu-22.04` so that we'd have full control of the version and remain one level back to help with user compatibility (in case users can't yet support ubuntu-24.04 images, more important for things like fabric-ca which uses cgo in the binary build). I realize that these changes are just for the runners and not the final user image so I'm fine going to ubuntu-24.04 in this repository. Just wanted to provide you the thinking from core fabric in case you want to remain consistent with `ubuntu-22.04`.

Reply:
24.04 is the current LTS so it seemed to make sense to shift to that instead of 22.04. I used an explicit Ubuntu release for tests and other things where it might be helpful to specify the Ubuntu version used for testing. For jobs where the version doesn't really matter, I figured ubuntu-latest meant they would stay current with minimal maintenance, and likely have access to the largest pool of hosted runners.